apps/testing-javascript/src/pages/playlists/index.tsx

```diff
@@ -4,12 +4,13 @@ import {type SanityDocument} from '@sanity/client'
 import {getAllPlaylists} from 'lib/playlists'
 import Link from 'next/link'
 import {track} from '@skillrecordings/skill-lesson/utils/analytics'
+import Playlist from 'components/playlist'
 
 export async function getStaticProps() {
   const playlists = await getAllPlaylists()
 
   return {
-    props: {modules: playlists.reverse()},
+    props: {modules: playlists},
```

Comment:
The modules were being listed in the wrong order, rely on the order they come in from Sanity